### PR TITLE
Some connected multimedia updates

### PR DESCRIPTION
**Pull request by satmandu**
Builds properly:
- [x] x86_64
- [x] armv7l
- [x] i686

### CODE REVIEW

File: packages/gst_plugins_bad.rb [hunk 1]
```diff
@@ -0,0 +1,66 @@
+require 'package'
+
+class Gst_plugins_bad < Package
+  description 'Multimedia graph framework - bad plugins'
+  homepage 'https://gstreamer.freedesktop.org/'
+  @_ver = '1.18.3'
+  version @_ver
```
Comment (uberhacker) [inline packages/gst_plugins_bad.rb:7]: Missing `compatibility 'all'`?

File: packages/gst_plugins_bad.rb [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+      -Dopenh264=disabled \
+      -Dopenmpt=disabled \
+      -Dopenni2=disabled \
+      -Dopensles=disabled \
+      -Dtinyalsa=disabled \
+      -Dvoaacenc=disabled \
+      -Dvoamrwbenc=disabled \
+      -Dwasapi2=disabled \
+      -Dwasapi=disabled \
+      -Dgobject-cast-checks=disabled \
```
Comment (uberhacker) [inline packages/gst_plugins_bad.rb:57]: What is the point of disabling everything?

File: packages/gst_plugins_good.rb [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  depends_on 'wavpack'
+  depends_on 'aalib'
+  depends_on 'taglib'
+  depends_on 'libdv'
+  # depends_on 'libshout'
+  depends_on 'libvpx'
+  depends_on 'jack' => ':build'
+  depends_on 'gtk3' => ':build'
+  # depends_on 'qt5_{tools,declarative,x11extras,wayland}' => ':build'
+  depends_on 'nasm' => ':build'
```
Comment (uberhacker) [inline packages/gst_plugins_good.rb:38]: Can we get rid of the commented `depends_on` lines?

File: packages/jack.rb [hunk 4]
```diff
@@ -32,16 +33,16 @@ def self.patch
...
+    system "env CFLAGS=-fno-stack-protector \
+      ./waf configure \
+      --dbus \
+      --classic \
+      --db=yes \
+      --alsa=yes \
+      --sndfile=yes \
+      --autostart=none \
+      --prefix=#{CREW_PREFIX} \
+      --libdir=#{CREW_LIB_PREFIX}"
```
Comment (uberhacker) [inline packages/jack.rb:45]: I assume you already tried `#{CREW_OPTIONS}`?_

Comment (satmandu) [reply packages/jack.rb:45]: Usually it's the target line in CREW_OPTIONS that manages to gum things up.

File: packages/gst_plugins_bad.rb (hunk 2 above)
Comment (satmandu) [reply packages/gst_plugins_bad.rb:57]: Cribbing from the working ARCH pkgbuild: https://github.com/archlinux/svntogit-packages/blob/packages/gst-plugins-bad/trunk/PKGBUILD

File: packages/gst_plugins_bad.rb (hunk 1 above)
Comment (satmandu) [reply packages/gst_plugins_bad.rb:7]: Good Catch.

File: packages/gst_plugins_good.rb (hunk 3 above)
Comment (satmandu) [reply packages/gst_plugins_good.rb:35]: done

File: packages/gst_plugins_bad.rb [hunk 5]
```diff
@@ -0,0 +1,67 @@
+require 'package'
+
+class Gst_plugins_bad < Package
+  description 'Multimedia graph framework - bad plugins'
+  homepage 'https://gstreamer.freedesktop.org/'
+  @_ver = '1.18.3'
+  version @_ver
+  compatibikity 'all'
```
Comment (uberhacker) [inline packages/gst_plugins_bad.rb:8]: Misspelled compatibility?

Comment (satmandu) [reply packages/gst_plugins_bad.rb:8]: Yup. This explains my poor second-place finish in that one elementary school spelling bee.

**Pull request merged**